src/ck-core/ckcallback.h

```diff
@@ -295,6 +295,18 @@ class CkCallback {
           d.group.refnum = 0;
 	}
 
+  void transformBcastToLocalElem() {
```

Comment:
Perhaps this can be converted to:
`void transformBcastToLocalElem(int elem = -1) {`
Then, do something like:
`d.group.onPE = (elem == -1) ? CkMyPe() : elem`
and, same with:
`d.group.onPE = (elem == -1) ? CkMyNode() : elem`

Just in the off-chance they want to route it elsewhere.